docs/rules/no-cycle.md

```diff
@@ -75,6 +75,22 @@ import { a } from './dep-a.js' // not reported as this module is external
 
 Its value is `false` by default, but can be set to `true` for reducing total project lint time, if needed.
 
+#### `allowUnsafeDynamicCyclicDependency`
+
+This option disable reporting of errors if a cycle is detected with at least one dynamic import.
```

Comment:
the intention, i thought, was that this option would not warn on cycles where the cycle passes through a dynamic import. if the cycle is separately discovered statically, then it should still warn on it.

Reply:
If I understand correctly, that's the current behavior. See the tests

Reply:
Test cases & fix have been added in 0fe7fc18336bba59398ff693d2e7012102885ae2.

Reply:
Up